wicket-core/src/test/java/org/apache/wicket/markup/html/basic/RedirectPage-expected1.html

```diff
@@ -0,0 +1,5 @@
+<html>
```

Comment:
why these new .html files are needed now ?

Reply:
They were needed because the core stylesheet was added, but now they are not really needed anymore. These tests use the same markup as input and expected result. Is this something we would like to keep? The problem I see with that is that regenerating the expected result actually changes the input of the test. That's why I kept these files.

Reply:
I see. Keep them!